### ISSUE TITLE
[ERROR] Line 26: The type com.google.gwt.user.client.ui.Accessibility is not visible

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Hilbrand/cobogw#4]
```
What steps will reproduce the problem?
1. Download the latest 1.2.2 release
2. Add a button to your GWT page
3. Try to compile your project

What is the expected output? What do you see instead?
I would expect it to compile, instead it complains about this class:
[ERROR] Line 26: The type com.google.gwt.user.client.ui.Accessibility is not 
visible

What version of the product are you using? On what operating system?
GWT 1.5, cobogw 1.2.2

Please provide any additional information below.

```

Original issue reported on code.google.com by `rustysh...@gmail.com` on 13 Mar 2009 at 4:39
